protobuf/drand/client.proto

```diff
@@ -68,3 +67,23 @@ message ECIESObject {
     bytes ciphertext = 2;
     bytes nonce = 3;
 }
+
+
+service Info {
+  rpc DistKey(DistKeyRequest) returns (DistKeyResponse) {
+    option (google.api.http) = {
+      get: "/public"
```

Comment:
I'm not sure of what it does when you put twice the same endpoint (for public randomness and public key). I would be much relieved if we just go with our own separation domain such as `/info/dist_public` (because `/info/public` returns another public key, his longterm public key)

Reply:
So you're thinking of something like 
`  rpc DistKey(DistKeyRequest) returns (DistKeyResponse) {
    option (google.api.http) = {
      get: "/dist_key"
    };
  }`
?

Reply:
So you're thinking of something like
`rpc DistKey(DistKeyRequest) returns (DistKeyResponse) { 
option (google.api.http) = { 
get: "/dist_key" 
};
}`
?